osu.Game.Rulesets.Mania/Difficulty/ManiaPerformanceCalculator.cs

```diff
@@ -34,82 +33,43 @@ protected override PerformanceAttributes CreatePerformanceAttributes(ScoreInfo s
             countGood = score.Statistics.GetValueOrDefault(HitResult.Good);
             countOk = score.Statistics.GetValueOrDefault(HitResult.Ok);
             countMeh = score.Statistics.GetValueOrDefault(HitResult.Meh);
             countMiss = score.Statistics.GetValueOrDefault(HitResult.Miss);
-
-            if (maniaAttributes.ScoreMultiplier > 0)
-            {
-                // Scale score up, so it's comparable to other keymods
-                scaledScore *= 1.0 / maniaAttributes.ScoreMultiplier;
-            }
+            scoreAccuracy = customAccuracy;
 
             // Arbitrary initial value for scaling pp in order to standardize distributions across game modes.
             // The specific number has no intrinsic meaning and can be adjusted as needed.
-            double multiplier = 0.8;
+            double multiplier = 8.0;
 
             if (score.Mods.Any(m => m is ModNoFail))
                 multiplier *= 0.9;
             if (score.Mods.Any(m => m is ModEasy))
                 multiplier *= 0.5;
 
             double difficultyValue = computeDifficultyValue(maniaAttributes);
-            double accValue = computeAccuracyValue(difficultyValue, maniaAttributes);
-            double totalValue =
-                Math.Pow(
-                    Math.Pow(difficultyValue, 1.1) +
-                    Math.Pow(accValue, 1.1), 1.0 / 1.1
-                ) * multiplier;
+            double totalValue = difficultyValue * multiplier;
 
             return new ManiaPerformanceAttributes
             {
                 Difficulty = difficultyValue,
-                Accuracy = accValue,
-                ScaledScore = scaledScore,
                 Total = totalValue
             };
         }
 
         private double computeDifficultyValue(ManiaDifficultyAttributes attributes)
         {
-            double difficultyValue = Math.Pow(5 * Math.Max(1, attributes.StarRating / 0.2) - 4.0, 2.2) / 135.0;
-
-            difficultyValue *= 1.0 + 0.1 * Math.Min(1.0, totalHits / 1500.0);
-
-            if (scaledScore <= 500000)
-                difficultyValue = 0;
-            else if (scaledScore <= 600000)
-                difficultyValue *= (scaledScore - 500000) / 100000 * 0.3;
-            else if (scaledScore <= 700000)
-                difficultyValue *= 0.3 + (scaledScore - 600000) / 100000 * 0.25;
-            else if (scaledScore <= 800000)
-                difficultyValue *= 0.55 + (scaledScore - 700000) / 100000 * 0.20;
-            else if (scaledScore <= 900000)
-                difficultyValue *= 0.75 + (scaledScore - 800000) / 100000 * 0.15;
-            else
-                difficultyValue *= 0.90 + (scaledScore - 900000) / 100000 * 0.1;
+            double difficultyValue = Math.Pow(Math.Max(attributes.StarRating - 0.15, 0.05), 2.2) // Star rating to pp curve
+                                     * Math.Max(0, 5 * scoreAccuracy - 4) // Accuracy curve
```

Comment:
Maybe add a comment that no PP is given below 80% acc, and starting from 80% acc the curve gives 1/20th of total PP per additional 1% acc.